test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -211,17 +201,14 @@ class CloudFoundryDeployTest extends BasePiperTest {
             cfCredentialsId: 'test_cfCredentialsId',
             cfManifest: 'test.yml'
         ])
-
-        assertTrue(jscr.shell[1].contains('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"'))
-        assertTrue(jscr.shell[1].contains('cf push -f "test.yml"'))
-
+        // asserts
+        assertThat(jscr.shell, hasItem(containsString('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"')))
```

Comment:
unrelated to scope of PR, only another way of asserting --> should be done in a dedicate PR.

Reply:
needed to change assertion because of `assertTrue(jscr.shell[0].contains...`